docs/DEPLOYING.md

```diff
@@ -114,6 +114,12 @@ This will generate two new migration scripts. Fill the `*.up.sql` file with the
 migration you want to run and the `*.down.sql` file with a script that reverses
 the first script.
 
+After adding a migration, you must add its version number to
+`datastore::SUPPORTED_SCHEMA_VERSIONS` as Janus will refuse to run if it does
+not recognize the database schema version. You also must add a `#[case]`
```

Comment:
nit: the docs suggesting that a `#[case]` attribute needs to be created can now be dropped.